### ISSUE TITLE
[VAMC] In the spotlight 'Read more' links need more descriptive link text

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#24072]
## Describe the defect
On the [VAMC Health Services page](https://www.va.gov/boston-health-care/health-services/) in then 'In the spotlight' section, there are three cards with 'Read more' links. For screen reader users navigating via link in a rotor menu or tabbing through the page, the links may be read without the rest of the info on the card making it difficult to understand the link destination.  

### Recommendation
Add context to the link name.

1. Ideally by writing more descriptive visible link text. Change 'Read more' to ...
2. OR use hidden/screen reader only text to provide more context for screen reader users only. Use aria-label, aria-labelledby, or a hidden span. Ensure that the accessible link name starts with the text in the visible link text to help voice navigation users with activating the links. 

## To Reproduce
1. Go to https://www.va.gov/boston-health-care/health-services/
2. Scroll down to the 'In the spotlight' section and read it out using the screen reader of your choice while tabbing through
3. Note that all three links read out as 'Read more'
4. If your screen reader has a rotor, open it and navigate to the links list
5. Note that there are multiple links with the same name in the list going to different destinations

## AC / Expected behavior
- [ ] Links in the 'In the spotlight' section have unique, descriptive link text 

## Screenshots
The 'In the spotlight' cards:
<img width="781" height="431" alt="Image" src="https://github.com/user-attachments/assets/cd738b00-f422-4fca-9fc2-63331b9c77a6" />

VoiceOver rotor showing multiple links with the same link text:
<img width="953" height="936" alt="Image" src="https://github.com/user-attachments/assets/55bf25d3-45d4-4382-80b1-d41864d2d413" />

**Comment by qjac** [comment]
@r-bartlett-gsa, this ticket will need some design and content love to determine what the accessible and/or visible text should be for the links. There are different spotlights for different system health services, so that's something to keep in mind as well.
